src/transformer/mockDefiner/mockDefiner.ts

```diff
@@ -266,6 +292,20 @@ export class MockDefiner {
         return [];
     }
 
+    private _getRegisterMockInFile(sourceFile: ts.SourceFile): ts.Statement[] {
+        if (this._registerMockFactoryRegistrationsPerFile[sourceFile.fileName]) {
+            return this._registerMockFactoryRegistrationsPerFile[sourceFile.fileName]
+                .map((reg: { key: ts.Declaration; factory: ts.Expression }) => {
+                    const key: string = this._registerMockFactoryCache.get(reg.key);
+                    // const key: string = this._factoryCache.get(reg.key);
```

Comment:
I think you just forgot this line. If I understand the intention is good to have the registerFactoryCache separated so we can clean it when need it.

Reply:
removed, it was a leftover